test/check-application

```diff
@@ -555,6 +559,75 @@ class TestApplication(testlib.MachineCase):
         b.wait_text("#containers-containers tbody tr:contains('busybox:latest') > td:nth-child(3)", "")
         b.wait_text("#containers-containers tbody tr:contains('busybox:latest') > td:nth-child(4)", "")
 
+    def testCheckpointRestore(self):
+        b = self.browser
+        m = self.machine
+
+        # Set machine to cgroup1 and reboot
+        self.execute(True, 'grubby --update-kernel=ALL --args="systemd.unified_cgroup_hierarchy=0"; reboot')
+        m.wait_reboot()
+
+        self.login_and_go("/podman", authorized=True, superuser=True)
+        b.wait_present("#app")
+        self.filter_containers('all')
+
+        # Run a container
+        self.execute(True, "podman run -dit --name swamped-crate busybox sh; podman stop swamped-crate")
+        b.wait(lambda: self.execute(True, "podman ps --all | grep -e swamped-crate -e Exited"))
+
+        b.wait_present("#containers-containers")
+        b.wait_present('#containers-containers tr:contains("swamped-crate")')
+        b.click('#containers-containers tbody tr:contains("busybox:latest") td.listing-ct-toggle')
+
+        # Check that the restore option is not present (i.e. start is a regular button)
+        b.wait_present('#containers-containers tbody tr:contains("busybox:latest") + tr button.pf-c-button:contains(Start)')
+
+        # Start the container
+        b.click('#containers-containers tbody tr:contains("busybox:latest") + tr button:contains(Start)')
+        b.wait(lambda: b.text('#containers-containers tr:contains(swamped-crate) td:nth-of-type(7)') == 'running')
```

Comment:
This can be written simpler as `b.wait_in_text('#containers-containers tr:contains(swamped-crate)', 'running') but surely fine as is :)

Reply:
I'll be doing changes in the tests anyway, so I'll simplify it - it will also help me remember that the method exists when writing Cockpit tests in the future.